_includes/charts.html

```diff
@@ -1,8 +1,8 @@
   {% if page.url == '/' %}
-    {% assign data_url = site.data_url %}
+    {% assign data_prefix = 'live' %}
     {% assign entity = "government" %}
   {% else %}
-    {% assign data_url = page.data_url %}
+    {% assign data_prefix = page.slug %}
```

Comment:
The `/live` prefix will be prepended to URLs that embed chart data when the page URL is `/`. This is a bit crude, but it was basically the status quo, and should work consistently.